README.md

```diff
@@ -23,6 +23,11 @@ These instructions are intended for bot development only.
 
 👉 **No support is offered for self-hosting the bot!** 👈
 
+### Requirements
+* Node `v14.0.0` and above.
+* [MariaDB](https://mariadb.org/), a drop-in replacement for [MySQL](https://www.mysql.com/).
+  * Note: MySQL will **not** work.
```

Comment:
This feels unnecessary
```suggestion
* [MariaDB](https://mariadb.org/)
```

Reply:
It seemed to me because I've seen and received a lot of questions about why MySQL isn't working... could shorten it to maybe say:

* [MariaDB](https://mariadb.org/) (MySQL will **not** work)

Reply:
Sounds good to me